### PR TITLE
update release-1.24 config.toml for release 1.25

### PR DESCRIPTION
**Pull request by cathchu**
update release-1.21 config.toml for new release 1.25

Version were updated according to [latest patch releases](https://github.com/kubernetes/website/blob/main/content/en/releases/patch-releases.md)

/cc @kcmartin @sethmccombs @krol3 @didicodes @reylejano

### CODE REVIEW

**Comment by jihoon-seo** [comment]
Preview:
- https://deploy-preview-36179--k8s-v1-24.netlify.app/releases/
- https://deploy-preview-36179--k8s-v1-24.netlify.app/releases/version-skew-policy/

File: config.toml [hunk 1]
```diff
@@ -139,13 +139,13 @@ time_format_default = "January 02, 2006 at 3:04 PM PST"
...
 showedit = true
 
-latest = "v1.24"
+latest = "v1.25"
 
-fullversion = "v1.24.0"
+fullversion = "v1.24.4"
 version = "v1.24"
-githubbranch = "main"
+githubbranch = "v1.24.4"
```
Comment (sftim) [inline config.toml:146]: This doesn't look right.

**Review by sftim — COMMENTED** [review]
/lgtm cancel

File: config.toml (hunk 1 above)
Comment (reylejano) [inline config.toml:146]:
```suggestion
githubbranch = "main"
```

File: config.toml [hunk 2]
```diff
@@ -179,40 +179,40 @@ js = [
 ]
 
 [[params.versions]]
-fullversion = "v1.24.0"
+fullversion = "v1.25.0"
 version = "v1.24"
-githubbranch = "v1.24.0"
+githubbranch = "v1.25.0"
 docsbranch = "main"
```
Comment (sftim) [inline config.toml:185]:
```suggestion
docsbranch = "release-1.24"
```

File: config.toml (hunk 1 above)
Comment (reylejano) [inline config.toml:146]:
```suggestion
githubbranch = "release-1.24"
```

**Review by kcmartin — APPROVED** [review]

**Pull request merged**